crates/subspace-service/src/lib.rs

```diff
@@ -90,31 +133,48 @@ impl sc_executor::NativeExecutionDispatch for ExecutorDispatch {
     }
 }
 
-/// Subspace full client.
-pub type FullClient =
+/// Subspace-like full client.
+pub type FullClient<RuntimeApi, ExecutorDispatch> =
     sc_service::TFullClient<Block, RuntimeApi, NativeElseWasmExecutor<ExecutorDispatch>>;
+
+/// Full client using subspace-runtime.
+pub type SubspaceClient = FullClient<subspace_runtime::RuntimeApi, SubspaceExecutorDispatch>;
+
 type FullBackend = sc_service::TFullBackend<Block>;
 type FullSelectChain = sc_consensus::LongestChain<FullBackend, Block>;
 
 /// Creates `PartialComponents` for Subspace client.
 #[allow(clippy::type_complexity)]
-pub fn new_partial(
+pub fn new_partial<RuntimeApi, ExecutorDispatch>(
```

Comment:
I would have preferred to move the file in one commit and then changing in another, otherwise it is unnecessarily hard to understand why something was renamed and refactored all at once.

Reply:
Make sense, but since it's already done, ...